ray-operator/apis/ray/v1/rayjob_types.go

```diff
@@ -59,6 +59,10 @@ const (
 	HTTPMode   JobSubmissionMode = "HTTPMode"   // Submit job via HTTP request
 )
 
+type SubmitterConfig struct {
+	SubmitterBackoffLimit *int32 `json:"submitterBackoffLimit,omitempty"`
```

Comment:
because this field is already embeeded within a `SubmitterConfig` I think this can just be:
```
BackOffLimit *int32 `json:"backoffLimit,omitempty"`
```

In YAML it would be cleaner as `spec.submitterConfig.backOffLimit` instead of `spec.submitterConfig.submitterBackOffLimit`